clients/drcachesim/drcachesim.dox.in

```diff
@@ -734,6 +734,13 @@ To invoke the miss analyzer, pass \p miss_analyzer to the \p -simulator_type
 parameter. To write the prefetching hints to a file use the \p -LL_miss_file
 parameter to specify the file's path and name.
 
+For example, to run the analyzer on a benchmark called "my_benchmark" and store
```

Comment:
Why not point the reader to the stride_benchmark?

Reply:
Updated.